lib/ansible/modules/network/iosxr/iosxr_user.py

```diff
@@ -83,6 +83,14 @@
         `admin` user and the current defined set of users.
     type: bool
     default: false
+  admin:
+    description:
+      - Enters into administration configuration mode for making config
+        changes to the device.
+      - Applicable only when using network_cli transport
+    type: bool
+    default: false
+    version_added: "2.8"
```

Comment:
@NilashishC can you give a related example under examples, so that user may get to know how to use the respective admin config mode.